src/App.js

```diff
@@ -15,7 +15,7 @@ function App() {
           target="_blank"
           rel="noopener noreferrer"
         >
-          Learn React
+          Learn GitHub
```

Comment:
Consider using this:

```suggestion
          You already know GitHub
```